source/tutorials/php.rst

```diff
@@ -0,0 +1,128 @@
+.. _php:
+
+Installing and Configuring PHP
+******************************
+
+This tutorial describes how to configure and use PHP and PHP extensions on |CL-ATTR|. Although we specifically address PHP this tutorial can serve as a general guide for working with applications in the |CL| :ref:`stateless` environment.
+
+We will focus on enabling PHP and PHP extensions in |CL|.  For a complete :abbr:`LAMP (Linux, Apache, MySQL, PHP)` setup in |CL| refer to the :ref:`web-server-install` section of the :ref:`wordpress` tutorial. For this tutorial you do not need to install MariaDB or WordPress.
+
+.. contents::
+    :local:
+    :depth: 1
+
+
+Prerequisites
+*************
+
+* :ref:`Install <bare-metal-install-desktop>` |CL| on your host system
+* Follow :ref:`web-server-install` to install Apache\* and PHP
+
+
+Configuring PHP
+***************
+
+.. important::
+
+   This tutorial does not cover configuring the php-fpm service. There is a difference between php and php-fpm here. php-fpm sets configuration for the process management and pools of worker bees, and can optionally set some of the values defined in the :file:`php.ini` file. This tutorial is only looking at the PHP configuration directives, not those of php-fpm.
+
+
+By default PHP looks for configuration settings in the :file:`php.ini` file, which resides in the `usr/share/defaults/php/` path. Because |CL| is designed to be a :ref:`stateless` operating system, you must create an optional configuration file to override the default values. Every time :command:`swupd` updates the system it will overwrite changes to the `/usr/share/defaults` file structure.
+
+
+Beginning with |CL| version 31020, PHP has been modified to check for :file:`.ini` files in the /etc/php.d file structure.
+
+You can verify the version of |CL| with :command:`swupd info`
+
+.. code-block:: bash
+
+    sudo swupd info
+
+You will see output similar to:
+
+.. code-block:: console
+
+   Distribution:      Clear Linux OS
+   Installed version: 31310
+   Version URL:       https://cdn.download.clearlinux.org/update
+   Content URL:       https://cdn.download.clearlinux.org/update
+
+
+You can create a :file:`php.ini` as follows:
+
+.. code-block:: bash
+
+   sudo mkdir -p /etc/php.d
+   sudo touch /etc/php.d/php.ini
```

Comment:
You might consider having the user-defined php configuration files named something different than `php.ini` just so that it is explicitly clear to the reader that the system php.ini is different from the users. 

It can be confusing for new readers because people tend to refer to any php configuration file as a "php-ini" even if the file isn't named php.ini.

Reply:
I've chosen the original "my-" prefix, so now the custom ini file is named "my-php.ini" :)